README.md

```diff
@@ -3,657 +3,288 @@
+1. start Eden's components - `eden start`
+   * if running EVE as qemu, entirely under eden control, it will start automatically
+   * if running EVE as a separate EVE device:
+     1. create or download the EVE image you want
+     1. flash the image to the device's storage
+     1. start the device
+1. onboard EVE - `eden eve onboard`, explicitly allowing it to connect to the controller
+1. use the Eden CLI to perform tasks, such as install apps or run tests
+1. terminate Eden's components and, optionally, EVE - `eden stop`
+1. clean up - `eden clean`
 
-Also, you need to install `telnet` and `squashfs-tools` (`squashfs` for Mac OS X).
+To customize the device on which EVE is running, or the image launched, see
+[docs/eve-images.md](./docs/eve-images.md).
 
-If you want to use Eden with RPI on Linux, you also need to install
-`binfmt-support` and `qemu-user-static`.
+### Prerequisites
 
-You need to be able to run docker commands and able to access virtualization accelerators
-(KVM on Linux or machyve on Mac OS X).
-For Linux, you need to add current user into docker and kvm groups:
+On the manager, you need:
 
-```console
+* the `eden` binary
+* [docker](https://docker.com), to run eden's components, including the ability to execute commands. For Linux:
+  ```console
 sudo usermod -aG docker $USER
 newgrp docker
-sudo usermod -aG kvm $USER
-newgrp kvm
-```
-
-If you want to be able to launch VMs or containers-in-VMs on Mac
-You need to use Parallels. ([Parallels Manual](./docs/parallels.md))
+  ```
+* test binaries - eden ships with pre-compiled ones
+* a text editor to configure the system and create test scenarios
+and scripts using `eden`
 
-## Quickstart Local (running on a PC in qemu mode)
+Eden itself -- the main executable file `eden`,
+components, and tests -- ships as stand-alone applications
+or docker images. You do not need to install and configure the development
+environment on your computer.
 
-Starts nginx Dockerhub image on port 8028 serving ./data/helloeve/ content, so you will be able to access it via `http://<EVE IP>:8028`. Feel free to change ./data/helloeve/ and redeploy the pod.
+The manager hosts:
 
-Eve Ip is displayed by `eden status` command
+* Eden - control layer CLI
+* Adam - the controller, running as a daemon process
+* Redis - the log database, running as a daemon process
+* Eserver - the image/file database to expose http and ftp content for downloading to the edge device
+* Registry - an OCI-compliant registry to expose images for downloading to the edge device
 
-```console
-git clone https://github.com/lf-edge/eden.git
-cd eden
-make clean
-make build-tests
-./eden config add default
-./eden setup
-source ~/.eden/activate.sh
-./eden start
-./eden eve onboard
-./eden pod deploy docker://nginx -p 8028:80 --mount=src=./data/helloeve,dst=/usr/share/nginx/html
-./eden status
-```
+Alternatively, Eden supports using a commercial controller.
```

Comment:
Not for now. Actually Eden can create an image for onboarding into commercial controller, but all other things are out of scope.

Reply:
Yeah, but that is too complex for the README. I fixed it. Take a look as soon as I push.